specs/latest/1.0/webgl.idl

```diff
@@ -55,6 +61,7 @@ dictionary WebGLContextAttributes {
     WebGLPowerPreference powerPreference = "default";
     boolean failIfMajorPerformanceCaveat = false;
     boolean desynchronized = false;
+    WebGLPixelFormat pixelFormat = "implicit";
```

Comment:
I might prefer having the key be "glPixelFormat" instead of "pixelFormat" to clearly mark that it is not part of 2D canvas. (But I'm not an authority on the particulars of this issue).

Reply:
I think we make people type gl enough haha!
I think if we added a different interpretation for canvas2d, that'd be ok. We can even make them alias if e.g. rgba8 and RGBA8 end up being the same, maybe.